src/website/views/components/expansion-panel.pug

```diff
@@ -115,7 +115,7 @@ p.-text
         </div>
       </div>
 
```

Comment:
Looks like using `h4` in expansion-panel like this causes issues. If h3 is not present before this component we get a heading skipping error. Which means that we will have the same problem with other components that use heading tags.

Reply:
@janibolkvadze good catch but I think we can solve in another PR just by changing h4 by a div

Reply:
@janibolkvadze @jllr This would have required changing the component so I created a separate ticket to solve for modifying the Expansion Panel component - PE-3849